app/gui/src/dashboard/providers/TextProvider.tsx

```diff
@@ -70,18 +68,12 @@ export default function TextProvider(props: TextProviderProps) {
 /** Exposes a property to get localized text, and get and set the current language. */
 export function useText() {
   const { language, setLanguage, locale } = React.useContext(TextContext)
-  const localizedText = text.TEXTS[language]
+
+  const localizedText = text.getDictionary(language)
 
   const getText = React.useCallback<GetText>(
```

Comment:
This function is pulled off the provider, so we can now use it outside of React.